src/styles/index.css

```diff
@@ -0,0 +1,248 @@
+body {
+  margin: 0;
+  font-family: Roboto, 'Times New Roman', serif;
+}
+
+.container {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  justify-content: center;
+  gap: 23px 24px;
+}
+
+@media (min-width: 488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --column-count: 4;
+  }
+}
+
+/* ====== Header ====== */
+.header {
+  display: flex;
+  height: 60px;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding: 3px 0 0;
+}
+
+.menu {
+  font-size: 12px;
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
```

Comment:
The `.menu` class should have a `display: flex;` property to ensure that its children (the list items) are laid out as flex items. This is missing here, which might cause layout issues.